.idea/vcs.xml

```diff
@@ -8,18 +8,6 @@
       <inspection_tool class="SubjectLimit" enabled="true" level="WARNING" enabled_by_default="true" />
     </profile>
   </component>
-  <component name="GithubSharedProjectSettings">
```

Comment:
Are these related changes/removals?

Reply:
Oops! Somehow my IDE insists changing this. Let me revert it.